cves/CVE-2018-6621.yml

```diff
@@ -64,7 +64,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  CVE-2018-6621 forces a video bytestream to extend past the normally allowed 
+  amount, resulting in an out-of-bounds read, a crash, and thus a denial of 
+  service. This vulnerability can be exploited with a maliciously crafted AVI 
```

Comment:
Maybe mention what an AVI file is. I know it is a commonly used term, but it doesn't hurt to explain it.

Reply:
I think it would be redundant and waste people's time if I were to explain what this particular file format was.